agent/config/muzero_2048_config.py

```diff
@@ -0,0 +1,92 @@
+from easydict import EasyDict
+
+
+# ==============================================================
+# begin of the most frequently changed config specified by the user
+# ==============================================================
+env_id = 'game_2048'
+action_space_size = 4
+collector_env_num = 8
+n_episode = 8
+evaluator_env_num = 3
+num_simulations = 100
+update_per_collect = 200
+batch_size = 512
+max_env_step = int(5e6)
+reanalyze_ratio = 0.
+num_of_possible_chance_tile = 2
+chance_space_size = 16 * num_of_possible_chance_tile
+# ==============================================================
+# end of the most frequently changed config specified by the user
+# ==============================================================
+
+atari_muzero_config = dict(
+    exp_name=f'data_muzero/game_2048_npct-{num_of_possible_chance_tile}_muzero_ns{num_simulations}_upc{update_per_collect}_rer{reanalyze_ratio}_bs{batch_size}_sslw2_seed0',
```

Comment:
simplify name here, this repo does not need too many hyper-parameters adjustment.